imagecraft/pack/gptutil.py

```diff
@@ -106,7 +106,8 @@ def _create_gpt_layout(
         start += sectors
     stdin_lines: str = "\n".join(_create_sfdisk_lines(header, partitions))
 
-    emit.debug(f"Stdin (sfdisk):\n{stdin_lines}")
+    emit.trace(f"Stdin for sfdisk:\n{stdin_lines}")
+    emit.progress("Partition the image")
```

Comment:
Present tense for progress?
```suggestion
    emit.progress("Partitioning the image")
```